packages/calling/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -48,35 +106,19 @@ describe('CallingClient Tests', () => {
      * calling and empty domain value.
      */
     it('Verify valid calling serviceData with no input sdk config', async () => {
-      expect(() => {
-        callingClient = createClient(webex);
+      expect(async () => {
+        callingClient = await createClient(webex);
       }).not.toThrow(Error);
-      expect(callingClient).toBeTruthy();
```

Comment:
Why was this line being removed everywhere?

Reply:
Reverted this removal.